include/sdsl/csa_alphabet_strategy.hpp

```diff
@@ -542,6 +542,173 @@ void init_char_bitvector(sd_vector<t_hi_bit_vector, t_select_1, t_select_0> &cha
     char_bv = std::move(sd_vector<t_hi_bit_vector, t_select_1, t_select_0>(builder));
 }
 
+/*!\brief Provides an alphabet mapping that implements an identity map (i.e. each character is mapped to its rank).
+ * \details This mapping is faster for FM indices and should always be used for ranges containing all characters of the
+ *          underlying alphabet type. Indices based on a text not containing all characters of its alphabet type will
+ *          have a much higher memory footprint using this alphabet mapping.
+ */
+class plain_byte_alphabet
+{
+public:
+    //! Helper class for the char2comp and comp2char mapping.
+    class mapping_wrapper;
+
+    typedef int_vector<>::size_type size_type;
+    typedef mapping_wrapper char2comp_type;
+    typedef mapping_wrapper comp2char_type;
+    typedef int_vector<64> C_type;
+    typedef uint16_t sigma_type;
+    typedef uint8_t char_type;
+    typedef uint8_t comp_char_type;
+    typedef std::string string_type;
+    typedef byte_alphabet_tag alphabet_category;
+    enum { int_width = 8 };
+
+    //! Helper class for the char2comp and comp2char mapping.
+    class mapping_wrapper
+    {
+    public:
+        //! Default constructor.
+        mapping_wrapper() = default;
+
+        //! Random access operator.
+        constexpr char_type operator[](char_type const c) const noexcept
+        {
+            return c;
+        }
+    };
+
+    const char2comp_type char2comp;
```

Comment:
```suggestion
    const char2comp_type char2comp{};
```

Fixes the clang-3.8 error

Reply:
> Fixes the clang-3.8 error

Thanks!